pract/index.html

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
```

Comment:
Hey @jainpatel25 
What is `pract/` directory?
You need to add your code for html in `index.html`, styles in 'assets/css/style.css' and JavaScript in `assets/javascript/app.js`